### PR TITLE
GH-1994: As a smith I want to extend and internally publish the n4jsd-generator

### PR DESCRIPTION
**Pull request by mor-n4**
See #1994.

### CODE REVIEW

File: n4js-tools/n4jsd-generator/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "n4jsd-generator",
+  "name": "@n4js-temp/n4jsd-generator",
```
Comment (mmews-n4) [inline n4js-tools/n4jsd-generator/package.json:2]: Seems that the parent folder `@n4js-temp` is missing.

File: releng/utils/scripts/publish-n4jsd-generator.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# use the following for excessive logging:
+#set -x
+
+
+NPM_REGISTRY="https://nexus3.internal.numberfour.eu/repository/npm-internal/"
+BASE_VERSION="0.0.1"
+
+# Choose a version for publishing
+TIMESTAMP_DATE=`date "+%Y%m%d"`
+TIMESTAMP_TIME=`date "+%H%M"`
```
Comment (mmews-n4) [inline releng/utils/scripts/publish-n4jsd-generator.sh:22]: maybe "time"?

**Review by mmews-n4 — APPROVED** [review]
👍  Nice to have that Jenkins!

**Comment by mor-n4** [comment]
Feedback implemented, thanks!

Merging ...

**Pull request merged**